### PR TITLE
Use more scss-isms in dark-theme

### PR DESCRIPTION
**Pull request by RubenRBS**
This allowed me to collapse some redundant rules, and by lexocigraphically sorting them they are now (I think) easier to maintain. There are no style changes

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -49,6 +49,11 @@
       <option name="ALIGN_MULTILINE_FOR" value="false" />
       <option name="SOFT_MARGINS" value="120" />
     </codeStyleSettings>
+    <codeStyleSettings language="SCSS">
+      <indentOptions>
+        <option name="INDENT_SIZE" value="4" />
```
Comment (RubenRBS) [inline .idea/codeStyles/Project.xml:54]: All our files use this

Comment (mattgodbolt) [reply .idea/codeStyles/Project.xml:54]: :+1:

**Review by mattgodbolt — APPROVED** [review]
Thanks!

**Comment by mattgodbolt** [comment]
Gonna merge to get this out

**Pull request merged**